scripts/Interface/reduction_application.py

```diff
@@ -283,14 +284,16 @@ def _change_instrument(self):
         """
             Invoke an instrument selection dialog
         """
+
         class InstrDialog(QDialog):
             def __init__(self, instrument_list=None):
                 QDialog.__init__(self)
                 self.ui = load_ui(__file__, 'ui/instrument_dialog.ui', baseinstance=self)
                 self.instrument_list = instrument_list
                 self.instr_combo.clear()
                 self.facility_combo.clear()
-                instruments = sorted(INSTRUMENT_DICT.keys())
+                instruments = sorted([fac for fac in INSTRUMENT_DICT.keys() if
```

Comment:
Can I be picky and ask us to rename the `instruments` variable to `facilities`? I was trying to understand the list comprehension assuming it was supposed to be giving a list of instruments and it didn't make sense until I realised it wasn't.